tests/acceptance/recipes/new-test.js

```diff
@@ -0,0 +1,27 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'recipe-card-19/tests/helpers/module-for-acceptance';
+
+moduleForAcceptance('Acceptance | recipes/new');
+
+test('User can visit a recipe page and see a recipe', function(assert) {
+  visit('/');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/');
+  });
+});
+
+test('User can see the serving size of the recipe', function(assert) {
+  let number = '.num-servings-input';
+  visit('/');
+
+  fillIn(number);
```

Comment:
`fillIn` expects two arugments, the selector that you fill in and the value that you want to fill in there.

Also this test is "User can see" so I would not fillIn until you have a test making sure the initial state of the app exists.